packages/fluent-ui/src/DateWidget/DateWidget.tsx

```diff
@@ -63,7 +63,7 @@ const DayPickerStrings: IDatePickerStrings = {
 const controlClass = mergeStyleSets({
   control: {
     margin: "0 0 15px 0",
-    maxWidth: "300px",
+    maxWidth: "700px",
```

Comment:
Why hardcode this width?

Reply:
To make the width match with DateTime field , in the original code the width was set to 300, I bumped it up to 700

Reply:
Removed the hard coded value